src/utils/registerDiscourseDatalogTranslators.ts

```diff
@@ -394,12 +394,14 @@ const registerDiscourseDatalogTranslators = () => {
                   dr.source === source ||
                   nodeLabelByType[dr.source] === source)
             );
+          pageNames.push(...Object.values(nodeLabelByType));
```

Comment:
If there is an existing page name in the graph called `Claim`, It will come up twice as an option.

we could avoid this by filtering the values before concatting to the array:

```typescript
...Object.values(nodeLabelByType).filter(p => sourcedRelations.some((sr) =>
     nodeByType[sr.target].text === p
));
```

this also removes a check for all the pages that it is not relevant for

Reply:
granted, this problem still exists if the `Claim` page itself happens to be a valid discourse node, but that doesn't seem likely.

To protect against that, we could do a `Array.from(new Set(array))` where `array` is our filtered page name list. but this isn't necessary and could be more expensive than its worth